from exceptions import CustomException

def will_throw_exception(request):
    raise CustomException()
